debian/control

```diff
@@ -43,8 +43,8 @@ Depends: cloud-guest-utils | cloud-utils,
          ${python3:Depends}
 Recommends: eatmydata, gdisk, gnupg, python3-apt, software-properties-common
 Suggests: openssh-server, ssh-import-id
-Replaces: cloud-init (<= 24.4~3+really24.3.1-0ubuntu4)
-Breaks: cloud-init (<= 24.4~3+really24.3.1-0ubuntu4)
+Replaces: cloud-init (<< 25.1.1-0ubuntu2~), cloud-init-base 
```

Comment:
Only diff I have is an additional commit to drop the trailing whitespace after cloud-init-base
```suggestion
Replaces: cloud-init (<< 25.1.1-0ubuntu2~), cloud-init-base
```